build.zig

```diff
@@ -892,6 +892,7 @@ fn maybe_execute(
 
         if (std.mem.eql(u8, arg, "--")) {
             build_and_run = true;
+            continue;
```

Comment:
Great catch!
It'd read better if we changed the next instruction to `if (build_and_run) to_run.append(arg)` instead of `if (!build_and_run) continue`.

Reply:
good idea